packages-ts/starknet-gauntlet-oz/src/commands/account/deploy.ts

```diff
@@ -47,7 +52,23 @@ const beforeExecute: BeforeExecute<UserInput, ContractInput> = (context, input,
 }
 
 const afterExecute: AfterExecute<UserInput, ContractInput> = (context, input, deps) => async (result) => {
-  deps.logger.success(`Account contract located at ${result.responses[0].tx.address}`)
+  const contract = result.responses[0].tx.address
+  contract
+    ? deps.logger.success(`Account contract located at ${contract}`)
+    : deps.logger.error('Account contract not successfully created')
```

Comment:
Hmm, `Account contract not successfully created` == `Account contract creation failed`? Or is there state in between?

Reply:
yup, it just means contract creation failed - I will make the msgs more explicit